monkey/infection_monkey/utils/commands.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from infection_monkey.model import CMD_CARRY_OUT, CMD_EXE, MONKEY_ARG
+
+LOG = logging.getLogger(__name__)
+
+
+def build_monkey_commandline(target_host, depth, vulnerable_port, location=None):
+    from infection_monkey.config import GUID
+
+    return "".join(
+        build_monkey_commandline_explicitly(
+            GUID,
+            target_host.default_tunnel,
+            target_host.default_server,
+            depth,
+            location,
+            vulnerable_port,
+        )
+    )
+
+
+def build_monkey_commandline_explicitly(
+    parent=None, tunnel=None, server=None, depth=None, location=None, vulnerable_port=None
+):
+    cmdline = []
+
+    if parent is not None:
+        cmdline.append("-p")
+        cmdline.append(f"{parent}")
```

Comment:
Personally, I would prefer using `str()` instead of format string literals for these. When you use `str()`, it [explicitly indicates](https://docs.python.org/3/library/stdtypes.html#str) that you want to convert something to a string. Using a format string _happens_ to convert these values to a string. When reading the code, I think `str(value)` more clearly indicates that the code's intent is to convert some data to a string.